console/src/components/UIKit/atoms/Button/Button.js

```diff
@@ -0,0 +1,46 @@
+import styled, { css } from 'styled-components';
+import {
+  layout,
+  space,
+  color,
+  border,
+  typography,
+  flexbox,
+} from 'styled-system';
+
+export const StyledButton = styled.button`
+  cursor: ${({ disabled }) => !disabled && 'pointer'};
+  appearance: button;
+
+  /* Pseudo classes like hover are not supported by Styled-System  */
+
+  ${({ type, theme, disabled, boxShadowColor }) => {
```

Comment:
Don't put JS into CSS. Move it outside of the CSS -- make React component.

Reply:
A React component is already there and I refactored it previously based on the last comment you left. Can you help me in this case and refactor it?

Reply:
Thanks :)